Install shared files in /usr/share/games/minetest-delta
User files go to ~/.config/minetest-delta
Games executables go to /usr/games
Place docs according to package name.
Index: minetest-delta-0.0.0+git20110719/CMakeLists.txt
===================================================================
--- minetest-delta-0.0.0+git20110719.orig/CMakeLists.txt	2011-07-19 19:40:56.786826152 +0200
+++ minetest-delta-0.0.0+git20110719/CMakeLists.txt	2011-07-19 19:46:17.456826260 +0200
@@ -53,10 +53,10 @@
 	set(DOCDIR "share/doc/minetest")
 	set(EXAMPLE_CONF_DIR ".")
 elseif(UNIX) # Linux, BSD etc
-	set(DATADIR "share/minetest")
-	set(BINDIR "bin")
-	set(DOCDIR "share/doc/minetest")
-	set(EXAMPLE_CONF_DIR "share/doc/minetest")
+	set(DATADIR "share/games/minetest-delta")
+	set(BINDIR "games")
+	set(DOCDIR "share/doc/minetest-delta")
+	set(EXAMPLE_CONF_DIR "share/doc/minetest-delta")
 endif()
 
 install(FILES "doc/README.txt" DESTINATION "${DOCDIR}")
Index: minetest-delta-0.0.0+git20110719/src/constants.h
===================================================================
--- minetest-delta-0.0.0+git20110719.orig/src/constants.h	2011-07-19 19:45:14.226826239 +0200
+++ minetest-delta-0.0.0+git20110719/src/constants.h	2011-07-19 19:45:30.616826244 +0200
@@ -28,7 +28,7 @@
 
 //#define HAXMODE 0
 
-#define APPNAME "minetest"
+#define APPNAME "minetest-delta"
 
 #define DEBUGFILE "debug.txt"
 
Index: minetest-delta-0.0.0+git20110719/src/porting.cpp
===================================================================
--- minetest-delta-0.0.0+git20110719.orig/src/porting.cpp	2011-07-19 19:41:02.386826156 +0200
+++ minetest-delta-0.0.0+git20110719/src/porting.cpp	2011-07-19 19:42:43.736826188 +0200
@@ -208,10 +208,10 @@
 	
 	pathRemoveFile(buf, '/');
 
-	path_data = std::string(buf) + "/../share/" + APPNAME;
+	path_data = std::string(buf) + "/../share/games/" + APPNAME;
 	//path_data = std::string(INSTALL_PREFIX) + "/share/" + APPNAME;
 	
-	path_userdata = std::string(getenv("HOME")) + "/." + APPNAME;
+	path_userdata = std::string(getenv("HOME")) + "/.config/" + APPNAME;
 
 	/*
 		OS X
